modules/charts/osuloader.js

```diff
@@ -0,0 +1,121 @@
+
+"use strict";
+
+const { floor } = require ('mathjs'); 
+const { Chart, Timepoint } = require("../chart.js");
+
+// add tag of data you want, colon is required
+const metadataTags = ["Title:","TitleUnicode:","Artist:","ArtistUnicode:","Creator:","Version:","BeatmapID:","HPDrainRate:","OverallDifficulty:"];
+
+const osuChart = new Chart({});
```

Comment:
This needs to be declared at the start of loadData so that we can use this script multiple times without reloading it.